README.md

```diff
@@ -48,14 +48,65 @@ Setup development environment
 ```
 
 Config file
-------------------------------------
+----------------------------------------------------------------------------------------------------------------------
 `config.json` file should be located in the same directory as all source files. Sample configuration is included in `config.json.sample` file.
 
 Possible fields in config file:
-
+- `authUrl` - URL to Keystone
+- `gohan` - Gohan connection config, must be an object with following fields:
+  - `schema` - path to JSON Schema
+  - `url` - URL to Gohan server
+- `sidebar` - array of custom sidebar buttons, must contain objects with following fields:
+  - `path`
+  - `title` 
+  - `id`
+- `sidebarFavorites` - array of links to views, must contain resources' IDs
+- `sidebarCategories` - array of objects including information about views' categories, must contain objects with following fields:
+  - `name` - display name of category
+  - `id` - ID of category
+  - `items` - array of links to views in the category
 - `sidebarChildResources` - array of schema IDs of child resources which should be visible in sidebar menu
 - `routes` - config array for routing, must contain objects with following fields:
   - `path` - URL path to view
   - `viewClass` - name of React component related to the view (in `src/routes/componentList.js`)
   - `name` - name of route
   - `exact` - match path exactly, default `true`
+- `errorMessages` - map of error messages
+  - `tokenExpire` - token expired message
+- `addingRelationDialog` - array of ????
```

Comment:
As far as I know this field isn't used in any of our UIs so IMO we can remove it from README, what do you think?